src/tests/ftest/io/mdtest.yaml

```diff
@@ -0,0 +1,60 @@
+hosts:
+    test_servers:
+        - boro-A
+        - boro-B
+        - boro-C
+        - boro-D
+    test_clients:
+        - boro-E
+        - boro-F
+timeout: 600
+server_config:
+    name: daos_server
+pool:
+    createmode:
+        mode_RW:
+             mode: 146
+    createset:
+        setname: daos_server
+    createsize:
+        scm_size: 15000000000
+    createsvc:
+        svcn: 1
+mdtest:
+    client_processes:
+        np_8:
+            np: 8
+    api: 'DFS'
+    test_dir: "/"
+    iteration: 1
+    data: !mux
+        zero:
+            write_bytes: 0
+            read_bytes: 0
+        4K:
+            write_bytes: 4000
```

Comment:
should be 4096 for both read and write (i think) to access to NVMe?

Reply:
Done